cluster/node-pools/master-default/userdata.yaml

```diff
@@ -599,11 +599,7 @@ write_files:
           effect: NoSchedule
         containers:
         - name: kube-controller-manager
-{{- if eq .Cluster.ConfigItems.kubernetes_controller_manager_image "zalando" }}
-          image: 926694233939.dkr.ecr.eu-central-1.amazonaws.com/production_namespace/teapot/kube-controller-manager-internal:v1.31.3-master-133
-{{- else }}
-          image: nonexistent.zalan.do/teapot/kube-controller-manager:fixed
-{{- end }}
+          image: nonexistent.zalan.do/teapot/{{if eq .Cluster.ConfigItems.kubernetes_controller_manager_image "zalando" }}kube-controller-manager-internal{{else}}kube-controller-manager{{end}}:fixed
```

Comment:
Is this intentional?

Reply:
Yes, @mikkeloscar added this patched version of the controller-manager. However, this should've been removed as the new k8s version we've rolled out doesn't require a patched controller manager 🤔

Reply:
Then it's your call, skipper side we can move forward.

Reply:
We can clean-up the patched controller manager as a follow-up.